src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,65 @@
+body {
+  margin: 0;
+  min-height: 100vh;
+
+  @include flex-justify-align(center, center);
+}
+
+.stopwatch {
+  width: 500px;
+  height: 500px;
+  background-color: aqua;
+  border-radius: 50%;
+  @include flex-justify-align(center, center);
+
+  &__hands {
+    width: 400px;
+    height: 400px;
+    position: relative;
+
+    &-circle {
```

Comment:
don't do this!
circle is not a part of __hands but independent element
It will be hard to find it by full name __hands-circle